requirements/app/base.txt

```diff
@@ -12,3 +12,4 @@ lightning-utilities==0.3.*
 beautifulsoup4<=4.8.2
 inquirer>=2.10.0
 psutil<5.9.4
+click>=8.0.0, <=8.1.3
```

Comment:
How come the app reqs didn't set this before? :thinking:

Reply:
Some other package must be using it so it gets installed alongside of it.